frontend/src/component/ui/Table.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Table } from './Table';
+
+describe('Table component', () => {
+  const sampleHeadings = ['Heading1', 'Heading2', 'Heading3'];
+  const sampleRows = [
+    ['row1-cell1', 'row1-cell2', 'row1-cell3'],
+    ['row2-cell1', 'row2-cell2', 'row2-cell3'],
+    ['row3-cell1', 'row3-cell2', 'row3-cell3']
+  ];
+
```

Comment:
I wonder what happens when some row has 2 columns and some has 10.
And what happens when the number of headers does not equal to the number of columns in some rows.

Reply:
It will look how any table with extra or less <td> tags look like. We might need to support `colspan` when the need arises.

![image](https://user-images.githubusercontent.com/18581705/78361287-c9106480-75d5-11ea-9141-93dea5e3b5e8.png)

Reply:
@rohithbalaji123 I think for header, we should consider uniform background or something similar. The header line seems being cut.

Reply:
Should we add a test case this?

Reply:
Sorry. I was just thinking maybe header background show have full width. But it seems that the backgrounds are added by the individual header themselves. So, it shouldn't be a problem here.